src/GregClient/GregClient.csproj

```diff
@@ -1,22 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-	<Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+	  <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <PackageId>Greg</PackageId>
     <Authors></Authors>
-    <Company>Autodesk</Company>
+    <Company>Autodesk, Inc</Company>
+    <AssemblyCopyright>Copyright � Autodesk, Inc 2022</AssemblyCopyright>
     <ProjectGuid>{644207B4-7E7F-474A-952E-3453960D8A01}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Greg</RootNamespace>
     <AssemblyName>Greg</AssemblyName>
     <TargetFrameworks>net8.0</TargetFrameworks>
-	  <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-	  <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <GenerateNuspecDependsOn>$(GenerateNuspecDependsOn);ReadPackageVersionFromOutputAssembly</GenerateNuspecDependsOn>
-	  <PackageIcon>logo_square_32x32.png</PackageIcon>
-	  <Description>The Dynamo Package Manager .Net Client.</Description>
-	  <OutputPath>..\..\bin\$(Configuration)\</OutputPath>
+    <TotalDays>$([System.DateTime]::op_Subtraction($([System.DateTime]::get_Now().get_Date()),$([System.DateTime]::new(2023,1,1))).get_TotalDays())</TotalDays>
+    <Days>$([MSBuild]::Multiply($(TotalDays), 10))</Days>
+    <Hours>$([MSBuild]::Divide($([System.DateTime]::get_Now().get_TimeOfDay().get_TotalHours()), 3))</Hours>
+    <Revision>$([MSBuild]::Add($(Days), $(Hours)).ToString('F0'))</Revision>
+    <Version>3.0.1.$(Revision)</Version>
+    <GeneratedAssemblyInfoFile>$(MSBuildProjectDirectory)/$(AppDesignerFolder)/AssemblyVersionInfo.cs</GeneratedAssemblyInfoFile>
```

Comment:
We're still using some MSBuild properties (and property functions), but these are accessible from the dotnet build command